application/actions/authsocial_test.go

```diff
@@ -39,12 +39,18 @@ func (ts *TestSuite) Test_addConfig() {
 		secret   string
 		want     map[string]SocialAuthConfig
 	}{
-		{name: "Facebook", authType: AuthTypeFacebook, key: fbKey, secret: fbSecret,
-			want: map[string]SocialAuthConfig{AuthTypeFacebook: fbConfig}},
-		{name: "Twitter", authType: AuthTypeTwitter, key: twKey, secret: twSecret,
-			want: map[string]SocialAuthConfig{AuthTypeTwitter: twConfig}},
-		{name: "Twitter", authType: AuthTypeLinkedIn, key: "", secret: twSecret,
-			want: map[string]SocialAuthConfig{}},
+		{
+			name: "Facebook", authType: AuthTypeFacebook, key: fbKey, secret: fbSecret,
```

Comment:
These probably would be more clean if split into more lines, but given their nature, I don't think it is necessary.